test/game_team_test.rb

```diff
@@ -29,7 +29,7 @@ def test_it_exists
   end
 
   def test_it_returns_details
-    assert_equal 2012030221, @game_team3.game_id
+    assert_equal "2012030221", @game_team3.game_id
     assert_equal 3, @game_team3.team_id
```

Comment:
changed the test to strings because the id's are listed as strings in the spec harness